pkgs/development/python-modules/rfcat/default.nix

```
@@ -0,0 +1,42 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, future
+, ipython
+, numpy
+, pyserial
+, pyusb
+, python
+}:
+
+buildPythonPackage rec {
+  pname = "rfcat";
+  version = "1.9.5";
+
+  src = fetchFromGitHub {
+    owner = "atlas0fd00m";
+    repo = "rfcat";
+    rev = "v${version}";
+    sha256 = "1mmr7g7ma70sk6vl851430nqnd7zxsk7yb0xngwrdx9z7fbz2ck0";
+  };
+
+  propagatedBuildInputs = [
+    future
+    ipython
+    numpy
+    pyserial
+    pyusb
+  ];
+
+  checkPhase = ''
+    ${python.interpreter} tests/*.py
+  '';
```

I think that we could work with the `pytestCheckHook` here. `pythonImportsCheck` is an additional safeguard.

```suggestion
  checkInputs = [
    pytestCheckHook
  ];

  pythonImportsCheck = [ "rflib" ];
```

---

Done. For Darwin, do you think I should include IOKit as well (since we added udev for Linux)? I was looking at other Python packages that rely on physical USB access but am not sure it is standard.

---

The package does not require it so for now I would just leave it out.